src/commands/connections/updateWebAppSettings.ts

```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AppSettingsTreeItem } from "vscode-azureappservice";
+
+export async function updateWebAppSetting(appSettItem: AppSettingsTreeItem, appSettingToUpdate: string, value: string): Promise<void> {
+    await appSettItem.editSettingItem(appSettingToUpdate, appSettingToUpdate, value);
```

Comment:
IMO this isn't worth it's own file anymore. You only did it because you didn't want to copy the code in the future when you support storage right? But it's only two lines of code now, so copying seems fine

Reply:
I did it also because of a future using this method for others type of connections. For now, I'm planning to use it only for SQL databases and not sure that I actually will need this as the method, so it's not hard to put it inside of the CosmosDBTreeItem and move into the method back when it's really needed.